maintenance/wikia/PurgeWikiCacheEverywhere.php

```diff
@@ -0,0 +1,51 @@
+<?php
+require_once __DIR__ . '/../Maintenance.php';
+require_once( dirname( __FILE__ ) . '/../../includes/objectcache/BagOStuff.php' );
+require_once( dirname( __FILE__ ) . '/../../includes/objectcache/MemcachedBagOStuff.php' );
+require_once( dirname( __FILE__ ) . '/../../includes/objectcache/MemcachedPeclBagOStuff.php' );
+
+class PurgeWikiCacheEverywhere extends Maintenance {
+	public function __construct() {
+		parent::__construct();
+		$this->mDescription = 'Purges the WikiFactory ,Fastly cache and memcache of a single wiki';
+	}
+
+	public function execute() {
+		global $wgCityId, $wgMemc;
+
+		//config values
+		$memCacheAlternativeServer = $_ENV['MEMCACHED_ALTERNATIVE_SERVER'] ? $_ENV['MEMCACHED_ALTERNATIVE_SERVER'] : 'prod.twemproxy.service.res.consul';
```

Comment:
Why env variable and not script parameter? I can't find this variable defined in any env.